ReSwiftDemo/ViewControllers/LoginViewController.swift

```diff
@@ -0,0 +1,51 @@
+import UIKit
+import ReSwift
+
+class LoginViewController: UIViewController {
+
+    @IBOutlet weak var loginTextField: UITextField!
+    @IBOutlet weak var passwordTextField: UITextField!
+    @IBOutlet weak var activityIndicator: UIActivityIndicatorView!
+    @IBOutlet weak var loginButton: UIButton!
+    
+    var store: Store<AppState>!
```

Comment:
mam mieszane uczucia co do `!` ;)

Reply:
Normalnie byłoby bez, a zależności by trzeba przekazać przez init, ale w przypadku viewControllerow inicjalizowanych ze storyboarda tak się nie da :(

Reply:
Podsyłałem Tobie link do prezentacji z `Current` :) Przyznam, że bardzo mi się to sprawdza :) 

https://github.com/sloik/CoJezdzi/blob/master/CoJezdzi/AppFramework/_App/_Dependency/Environment.swift